docs/agent-config.adoc

```diff
@@ -0,0 +1,44 @@
+= Agent configuration
+
+Agent can be configured by running `hercules-ci-agent --config FILE.toml`
+config formatted using https://github.com/toml-lang/toml#toml[TOML] markup.
+
```

Comment:
A minimal example file would be nice, to demonstrate the syntax and as a starting point.